pom.xml

```diff
@@ -929,6 +1000,32 @@
         <profile>
             <id>zip</id>
             <modules>
+                <module>extensions</module>
+                <module>hazelcast</module>
+                <module>hazelcast-jet-sql</module>
+                <module>hazelcast-spring</module>
+                <module>hazelcast-spring-tests</module>
+                <module>hazelcast-build-utils</module>
+                <module>hazelcast-sql-core</module>
+                <module>hazelcast-sql</module>
+                <module>hazelcast-all</module>
+                <module>distribution</module>
+            </modules>
+        </profile>
+
+        <profile>
+            <id>quick</id>
+            <properties>
+                <checkstyle.skip>true</checkstyle.skip>
+            </properties>
+            <modules>
+                <module>hazelcast</module>
+                <module>hazelcast-jet-sql</module>
```

Comment:
The `hazelcast-jet-sql` requires some extension modules for running tests, so it would not be possible to run quick profile with tests (it would use stale version of those dependencies from your local repo).

But I would actually skip tests for the quick profile, also with javadoc.